src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,94 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
+import { Movie } from '../../types/Movie';
 import { TextField } from '../TextField';
 
-export const NewMovie = () => {
+type Props = {
+  onAdd: (movie: Movie) => void;
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImbId] = useState('');
+
+  const reset = () => {
+    setTitle('');
+    setDescription('');
+    setImgUrl('');
+    setImdbUrl('');
+    setImbId('');
+  };
+
+  const isDisabled = !(
+    title.trim() && imgUrl.trim() && imdbUrl.trim() && imdbId.trim
+  );
```

Comment:
You forgot to add brackets after trim, which is why your tests are failed
```suggestion
  const isDisabled = !(
    title.trim() && imgUrl.trim() && imdbUrl.trim() && imdbId.trim()
  );
```

Reply:
yeah, fix it